addons/website/static/src/xml/website.editor.xml

```diff
@@ -23,23 +23,99 @@
         </Dialog>
     </t>
 
-    <!-- Add a Google Font option dialog -->
-    <t t-name="website.dialog.addGoogleFont">
-        <Dialog title="this.props.title">
-            <div class="mb-3 row">
-                <label class="col-form-label col-md-3" for="google_font_html">Google Font address</label>
-                <div class="col-md-9">
-                    <textarea id="google_font_html" class="form-control o_input_google_font" t-ref="fontInput"
-                        placeholder="https://fonts.google.com/specimen/Roboto" style="height: 100px;"/>
-                    <span class="float-end text-muted">
-                        Select one font on <a target="_blank" href="https://fonts.google.com">fonts.google.com</a> and copy paste the address of the font page here.
-                    </span>
+    <!-- Add a Font option dialog -->
+    <t t-name="website.dialog.addFont.singlePreview">
+        <div class="row">
+            <label class="col-12" t-attf-style="font-weight: {{previewWeight}};" t-out="previewLabel"/>
+        </div>
+        <div class="mb-3 row">
+            <div class="col-11 offset-1" t-attf-style="font-family: '{{previewFontFamily}}';">
+                <input t-if="previewFontName" class="w-100 border-0" t-model="state.previewText" t-attf-style="font-weight: {{previewWeight}}; background-color: transparent;"/>
+                <input t-else="" class="w-100 border-0" readonly="1" t-attf-style="font-weight: {{previewWeight}}; background-color: transparent;"/>
```

Comment:
`t-model` is explained [here](https://github.com/odoo/owl/blob/master/doc/reference/input_bindings.md) - and is the reason why we need the `t-if`. It is used here so that if you edit the preview text in one style you get the same text in all previews.
For "normal" attributes, we could use `t-att-*` or `t-attf-*`, but this is already a "Qweb-level" attribute, and we cannot `t-att-t-model`.

Using `bg-transparent` now. :+1: